casper/src/main/scala/coop/rchain/casper/genesis/contracts/RevGenerator.scala

```diff
@@ -19,57 +19,63 @@ final case class RevGenerator(
 
   val code: String =
     s""" new rl(`rho:registry:lookup`), revVaultCh in {
-       #   rl!(`rho:rchain:revVault`, *revVaultCh)
-       #   | for (@(_, RevVault) <- revVaultCh) {
+       #   rl!(`rho:rchain:revVault`, *revVaultCh) |
+       #   for (@(_, RevVault) <- revVaultCh) {
+       #
        #     new genesisVaultCh in {
        #       @RevVault!(
```

Comment:
Now that vault construction is inlined, we can eliminate the genesis vault. Please remove it.